meta-xilinx-bsp/recipes-graphics/libgles/libmali-xlnx.bb

```diff
@@ -116,6 +116,12 @@ do_install() {
     fi
 }
 
+do_install_append () {
+    # libwayland-egl has been moved to wayland 1.15+
+    rm -f ${D}${libdir}/libwayland-egl*
+    rm -f ${D}${libdir}/pkgconfig/wayland-egl.pc
+    rmdir --ignore-fail-on-non-empty ${D}${libdir}/pkgconfig
+}
```

Comment:
Funny, I went for the opposite in my early wayland tests. I hacked the do_install of the wayland recipe instead. I started a patchset whereby I thought this lib should become virtual (so virtual/libwayland-egl) and added the `PROVIDES` and `PREFERRED_PROVIDER_virtual/libwayland-egl` ... I have not check which libwayland-egl was "better" at providing the code. Had to put that work aside for now.

One important failure of libmali-xlnx is that it doesn't provide the .h files as a provider of a library should in it's *-dev package.

Does anyone know if the libwayland-egl from libmali-xlnx is modified for zynqmp/mali and should be preferred over the default wayland source?